gcloud-java-cloudtrace/src/main/java/com/google/cloud/cloudtrace/spi/v1/TraceServiceApi.java

```diff
@@ -0,0 +1,422 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.cloudtrace.spi.v1;
+
+import com.google.api.gax.core.PageAccessor;
+import com.google.api.gax.grpc.ApiCallable;
+import com.google.api.gax.protobuf.PathTemplate;
+import com.google.devtools.cloudtrace.v1.GetTraceRequest;
```

Comment:
Same comments here as for `errorreporting`, shouldn't also this package be renamed?